app/models/problem_check_tracker.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class ProblemCheckTracker < ActiveRecord::Base
+  validates :identifier, presence: true, uniqueness: true
+  validates :blips, presence: true, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
This is not me trying to be cute. It was actually the most accurate name I could think of. 😅